pkg/proxy/ipvs/proxier.go

```diff
@@ -1547,17 +1556,27 @@ func (proxier *Proxier) syncProxyRules() {
 		klog.ErrorS(err, "Error syncing healthcheck endpoints")
 	}
 
+	metrics.SyncProxyRulesNoLocalEndpointsTotal.WithLabelValues("internal").Set(float64(proxier.serviceNoLocalEndpointsInternal.Len()))
+	metrics.SyncProxyRulesNoLocalEndpointsTotal.WithLabelValues("external").Set(float64(proxier.serviceNoLocalEndpointsExternal.Len()))
```

Comment:
are these metrics related to this PR?

Reply:
no, but they are tracked before conntrack cleaning in IPtables, hence moved their position here in ipvs proxier.

Reply:
then use independent commit for the metrics, if we want to backport the fix we may have conflicts with them